rocAL/rocAL_pybind/amd/rocal/plugin/pytorch.py

```diff
@@ -103,6 +105,8 @@ def __init__(self, pipeline, tensor_layout = types.NCHW, reverse_channels = Fals
                     self.out = torch.empty((self.bs*self.n, self.p, int(self.h/self.bs), self.w,), dtype=torch.float32, device = torch_gpu_device)
                 elif self.tensor_dtype == types.FLOAT16:
                     self.out = torch.empty((self.bs*self.n, self.p, int(self.h/self.bs), self.w,), dtype=torch.float16, device = torch_gpu_device)
+                elif self.tensor_dtype ==types.UINT8:
+                    self.out = torch.empty((self.bs*self.n, self.p, int(self.h/self.bs), self.w,), dtype=torch.uint8, device = torch_gpu_device)
                 torch_gpu_device = torch.device('cuda', self.device_id)
```

Comment:
looks like torch_gpu_device is defines again in this line. Please remove duplicate definitions (applies to other such cases)

Reply:
done